plugins/uploadwidget/plugin.js

```diff
@@ -300,8 +300,9 @@
 			 * @property {Function}
 			 * @param {String} data HTML to replace the upload widget.
 			 * @param {String} [mode='html'] See {@link CKEDITOR.editor#method-insertHtml}'s modes.
+			 * @param {Boolean} [skipSelection='false'] allows skipping data selection after upload widget replace.
```

Comment:
Rather than skipping selection and delegating whole logic, I'd think about implementing it directly in `uploadwidget` or even lower, in `widget`. This PR introduces selection preservation only or `uploadimage`, but exactly the same case is also present in e.g. Easy Image, not based on `uploadwidget`.